www/app/app.ts

```diff
@@ -470,11 +470,41 @@ export class App {
         });
   }
 
+  // If the provided url is actually an invite page, with the shadowsocks link in a URL
+  // fragment, this function extracts the shadowsocks link and returns it.  Otherwise,
+  // it returns the input unmodified.
+  private unwrapInvite(url: string): string {
+    try {
+      const invite: URL = new URL(url);
+      const baseURLs: string[] = [
+        'https://s3.amazonaws.com/outline-vpn/invite.html',
+        'https://s3.amazonaws.com/outline-vpn/index.html'
```

Comment:
Will `index.html` ever trigger?

Reply:
It will trigger if someone copies an "old" (i.e. current) invite link and then opens their Outline client.  (On some platforms they might have to paste too.)